src/dialects/sqlite3/schema/ddl.js

```diff
@@ -38,27 +46,31 @@ assign(SQLite3_DDL.prototype, {
     });
     if (!currentCol)
       throw new Error(
-        `The column ${column} is not in the ${this.tableName} table`
+        `The column ${column} is not in the ${this.tableName()} table`
       );
     return currentCol;
   }),
 
   getTableSql() {
-    return this.trx.raw(
-      `SELECT name, sql FROM sqlite_master WHERE type="table" AND name="${
-        this.tableName
-      }"`
-    );
+    this.trx.disableProcessing();
```

Comment:
this is an internal query which breaks if we e. g. uppercase fields name and sql.